### DISCUSSION TITLE
Is there a Copilot issue with Chromebook/VSCODE Linux? (CoPilot no working for me)

### DISCUSSION THREAD

**Discussion by rasaBakolong** [community/community#67992] [Programming Help] [answered]
Novice programmer here. 
So I want to try CoPilot: I  downloaded the extension, I have an active subscription, VSCODE is up to date, and everything is connected proper (no error messages). I'm still no getting any suggestions? I tried peeking in output and I'm getting this line repeatedly -> "[ghost Text] ... cancelled during debounce". 

What I've tried: 
1. Uninstalling/ Reinstalling Copilot
2. Restarting VSCODE / Restarting my Computer 
3. Disabled other extensions 

My Machine: Lenovo IP Slim 3 Chromebook
I'm running VSCODE Linux Debian

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by vishalkrishna8** [comment, marked as answer]
Hello @demBakolong  Please check is there any error connecting to the GitHub API to request a token to use GitHub Copilot.

**Reply by rasaBakolong** [reply]
I reconfigured the connection and it seems to be working ok now. Thank you.
